course_content/notebooks/numpy_intro.ipynb

```diff
@@ -1008,30 +1084,41 @@
    },
    "outputs": [],
    "source": [
-    "daily_records = np.array([[12, 14, 16], [11, 13, 15]])\n",
-    "masked_data = np.ma.masked_array(daily_records)\n",
-    "masked_data[0, 1] = np.ma.masked\n",
     "print 'unmasked average = ', np.mean(daily_records)\n",
-    "print 'masked average = ', np.ma.mean(masked_data)\n"
+    "print 'masked average = ', np.ma.mean(masked_data)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
-   "source": []
+   "source": [
+    "The `np.ma.masked_array()` function seen above is a simple creation method for masked data.  \n",
+    "The sub-module `np.ma` contains all the NumPy masked-data routines.\n",
+    "\n",
+    "Instead of masking selected points, as above, a mask is often specified as a \"mask array\":  This is a boolean array of the same size and shape as the data, where `False` means a good datapoint and `True` means a masked or missing point.  Such a 'boolean mask' can be passed in the `np.ma.masked_array` creation method, and can be extracted with the `np.ma.getmaskarray()` function.\n",
```

Comment:
Couldn't you achieve the same as using `np.ma.getmaskarray()` just with `ma.mask`?